mta/mta.sw.yaml

```diff
@@ -154,12 +155,16 @@ states:
           refName: createNotification
           arguments:
             title: "\"MTA analysis report for \" + .application.repository.url"
-            message: '"<a href=\"" + $SECRET.mta_url+ "/hub/applications/" + (.application.id|tostring)+ "/bucket/" + .data.output + "\">MTA Analysis Rerport</a>"'
+            message: '"<a href=\"" +  + "\">MTA Analysis Rerport</a>"'
             origin: "MTA Analysis Workflow"
             targetUsers:
               - .backstageUserOrGroup
             targetGroups:
               - .backstageUserOrGroup
+            actions:
+              - title: "MTA analysis report"
```

Comment:
pls replace with "View report" (this goes under action - hence the action form of the title)